### DISCUSSION TITLE
Example Ubuntu Devcontainer

### DISCUSSION THREAD

**Discussion by rcplane** [devcontainers/community#67] [Q&A]
Hello,

I'm trying to get started working on a public Github development project using Github Codespaces through VSCode on a Mac laptop, hoping to share our development environment via devcontainers.

Where is there a good getting started example with full code?
I've tried both guided feature installs to build a simple alpine image and an ubuntu devcontainer sample without success.
https://github.com/microsoft/vscode-dev-containers/blob/main/containers/ubuntu/.devcontainer/devcontainer.json
Error code: 1302 (UnifiedContainersErrorFatalCreatingContainer)

In the past I have developed in Docker locally and on cloud for more than five years, so I'm comfortable with the Docker build and VSCode Docker extension from Microsoft seems to be installed and running well.

Any advice would be appreciated, and I can try making a minimal reproduction example as a next step if there isn't a designated getting started example already around.

**Comment by samruddhikhandale** [comment]
Hi 👋 

As we have migrated to the `devcontainers` org, we will soon be archiving [microsoft/vscode-dev-containers](https://github.com/microsoft/vscode-dev-containers) repository. As that repo is no longer maintained, that could be the cause of `Error code: 1302 (UnifiedContainersErrorFatalCreatingContainer)`

@rcplane have you looked at [devcontainers/templates](https://github.com/devcontainers/templates)? From within a codespace, you could add any Template and/or Features to your repo with the following command 👇 See https://docs.github.com/en/codespaces/setting-up-your-project-for-codespaces/adding-a-dev-container-configuration/introduction-to-dev-containers#using-a-predefined-dev-container-configuration for more information.

<img width="716" alt="image" src="https://github.com/devcontainers/community/assets/24955913/39bdd055-30e1-4dac-a4e2-35697858826e">


Also, https://github.com/codespaces/templates has some codespaces templates to start from. See https://docs.github.com/en/codespaces/developing-in-codespaces/creating-a-codespace-from-a-template
